src/clang-c-frontend/clang_c_convert.cpp

```diff
@@ -2680,6 +2745,29 @@ bool clang_c_convertert::get_expr(const clang::Stmt &stmt, exprt &new_expr)
     break;
   }
 
+  case clang::Stmt::ChooseExprClass:
+  {
+    const clang::ChooseExpr &cexpr =
+      static_cast<const clang::ChooseExpr &>(stmt);
+
+    if (get_expr(*cexpr.getChosenSubExpr(), new_expr))
```

Comment:
Is it possible for `cexpr.getChosenSubExpr` to return a null ptr?

Reply:
No, it can't be. I've added the comment from the [Clang docs](https://clang.llvm.org/doxygen/classclang_1_1ChooseExpr.html#details).